src/main/java/lotto/domain/PrizeMoney.java

```diff
@@ -0,0 +1,23 @@
+package lotto.domain;
+
+public class PrizeMoney {
+
+    private long amount;
```

Comment:
클래스를 만들주실때는 final을 붙여서 불변 객체로 만들어주시면 좋습니다 ! :)

```suggestion
    private final long amount;
```